src/xbot/common/subsystems/pose/commands/SetRobotHeadingCommand.java

```diff
@@ -0,0 +1,35 @@
+package xbot.common.subsystems.pose.commands;
+
+import com.google.inject.Inject;
+
+import xbot.common.command.BaseCommand;
+import xbot.common.subsystems.pose.BasePoseSubsystem;
+
+public class SetRobotHeadingCommand extends BaseCommand {
+
+    BasePoseSubsystem poseSubsystem;
+    double heading;
+    
+    @Inject
+    public SetRobotHeadingCommand(BasePoseSubsystem poseSubsystem) {
+        this.poseSubsystem = poseSubsystem;
+        heading = BasePoseSubsystem.FACING_AWAY_FROM_DRIVERS;
+    }
+    
+    public void setHeadingToApply(double heading) {
```

Comment:
Yeah - this is primarily used for big static starting positions, so they don't need to be dynamically changed too much at runtime.